lib/polymorphic_integer_type/extensions.rb

```diff
@@ -20,37 +20,11 @@ def belongs_to(name, scope = nil, **options)
           _polymorphic_foreign_types << foreign_type
 
           # Required way to dynamically define a class method on the model
-          singleton_class.__send__(:define_method, "#{foreign_type}_mapping") do
+          define_singleton_method("#{foreign_type}_mapping") do
```

Comment:
just switching to more idiomatic way of defining singleton methods

Reply:
IIRC, `define_singleton_method` didn't exist in earlier versions of Ruby when we first wrote this module, and `__send__` was the only way to do it. This is much nicer to read.